radicle/src/node.rs

```diff
@@ -15,7 +15,7 @@ pub const DEFAULT_SOCKET_NAME: &str = "radicle.sock";
 
 #[derive(thiserror::Error, Debug)]
 pub enum Error {
-    #[error("failed to connecto to node: {0}")]
+    #[error("failed to connect to to node: {0}")]
```

Comment:
Lol, you fixed one type and introduced another :sweat_smile:

Reply:
I kind of like `connecto` :P

Reply:
hahahahahaahaha